### ISSUE TITLE
Mark mutually-exclusive menu items as type:radio in Electron Desktop

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#10876]
Main menu items such Debug / On Error contain multiple items where only one at a time can be checked.

![Screen Shot 2022-03-31 at 11 40 16](https://user-images.githubusercontent.com/10569626/161116815-1aef50ed-808c-4a31-9040-cf8fcfc8260f.png)

For accessibility we should mark these menu items as type:radio, not type:checkbox as we currently do. We decided to only do this if the accessibility switch is on (the appearance of radio menus is quite jarring; not seen very commonly, and really only beneficial to screen-reader users).

These are annotated as `radio` in command.cmd.xml: see https://github.com/rstudio/rstudio/commit/47fc0dd5 for when I hooked that up.

**Comment by timtmok** [comment]
I think the `radio` just might work. The real test will be the View > Panes > Console on left/right items.

**Comment by matheustavaresdev** [comment]
It works only when we click on it twice which is kind of weird...

**Comment by ronblum** [comment]
Tested with build RStudio Desktop 2022.06.0-daily-316. Marking as failed for the following reasons: 
- MacOS 12.4: Works as before, and no change expected—success
- Red Hat 8.4: Shows checks as in MacOS, instead of radio buttons—fail
- Ubuntu 20.04: Looks like a checkbox, instead of radio buttons—fail (unless this is the only option on Ubuntu)

Windows TBD due to #10162.

Red Hat Example: 
![radio_redhat.png](https://codetreeusercontent.s3.amazonaws.com/uploads/d10f0c30-dfea-433a-8825-88517054a697/radio_redhat.png)

Ubuntu Example:
![radio_ubuntu.png](https://codetreeusercontent.s3.amazonaws.com/uploads/d10f0c30-dfea-433a-8825-88517054a697/radio_ubuntu.png)

**Comment by gtritchie** [comment]
It is possible none of the Linux desktop managers support displaying these as checkboxes; we have no direct control over that.

So, if this works on Mac and Windows I'd consider it sufficient.

**Comment by gtritchie** [comment]
Build 2022.06.0-daily+333 for RHEL8 on Rocky Linux (with Gnome) is showing me the radio menus as expected. Did you have accessibility enabled when you tried? Or maybe using a different window manager like KDE Plasma (which I haven't tried)?

**Comment by gtritchie** [comment]
FYI, I installed Plasma on my Ubuntu 22 machine, and confirmed that menus don't support showing radio buttons on KDE, only on Gnome.

This is fine (and even if it wasn't there's nothing we could do about it).
